code/game/machinery/teleporter.dm

```diff
@@ -22,14 +22,9 @@
 	var/area_bypass = FALSE
 	var/cc_beacon = FALSE
 
-/obj/machinery/computer/teleporter/New()
-	src.id = "[rand(1000, 9999)]"
```

Comment:
The last commit for this line was the initial google code import of TG, and `link_power_station()` is already called in Initialize(), and the ID set isn't important to happen this early.